src/YieldDisburser.sol

```diff
@@ -125,12 +134,12 @@ contract YieldDisburser is OwnableUpgradeable {
      */
     function _castVote(uint256[] calldata percentages, address holder) internal {
         uint256 length = breadchainProjects.length;
-        require(percentages.length == length, "Incorrect number of projects");
+        if (percentages.length != length) revert IncorrectNumberOfProjects();
         uint256 total;
         for (uint256 i = 0; i < length; i++) {
             total += percentages[i];
         }
-        require(total == 100, "Total must equal 100");
+        require(total == 100, "Total must equal 100"); // TODO not creating custom error in anticipation of #16
```

Comment:
all the errors should be changed including this one. if they're not used in future changes, they can be removed then